circle.yml

```diff
@@ -21,12 +21,12 @@ test:
 
 deployment:
   production:
-    branch: master
+    branch: latest
```

Comment:
This changes the frequency of adding a build to https://hub.docker.com/r/interledgerjs/ilp-kit/tags/